docs/example-scenario/infrastructure/multi-tier-app-disaster-recovery.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Highly available (HA) and disaster recovery (DR) configured multi-tier web application
+description: Build a highly available and disaster recovery configured multi-tier web application on Azure using Azure virtual machines, Availability sets, Availability zones, Azure Site Recovery and Azure Traffic Manager
+author: sujayt
+ms.date: 11/16/2018
+---
+
+# Build a highly available and disaster recovery configured multi-tier web application
+
+This example scenario is applicable to any industry that have a need to deploy resilient multi-tier applications with high availability and disaster recovery configured. In this scenario, the application consists of 3 layers.
+
+- Web tier - This is the top most layer with user interface. This translates inputs and outputs into something user understands.
```

Comment:
This statement feels awkward: "This translates inputs and outputs into something user understands."

Reply:
changes to a different sentence

- Web tier - This is the top most layer with user interface. This layer parses the user interactions and passes the actions to next layer for processing.